pennylane/qchem/molecule.py

```diff
@@ -107,12 +107,6 @@ def __init__(
 
         self.n_electrons = sum(self.nuclear_charges) - self.charge
 
-        if self.n_electrons % 2 == 1 or self.mult != 1:
-            raise ValueError(
-                "Openshell systems are not supported. Change the charge or spin "
-                "multiplicity of the molecule."
-            )
-
```

Comment:
A little confused here 😅 
Does molecule.py support open-shell systems? Or not? The documentation suggests they are not supported but this deletion suggests they are supported... Is the error simply being moved elsewhere?